manifests/stage-ofed-driver/0100_service_account.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
```

Comment:
can you provide some info about why is that needed ?

Reply:
also is it needed per stage ? i see you added it just for ofed-driver stage.

Reply:
In order to add the SecurityContextConstraints (SCC) below, we need the corresponding ServiceAccount, etc.